### DISCUSSION TITLE
Vevői app 2.

### DISCUSSION THREAD

**Discussion by MicraMetripond** [nav-gov-hu/eRECEIPT#23] [Kérdések és válaszok]
Ha újra kell telepíteni a vevő app-ot, akkor az, hogy fogja biztosítani, hogy minden új vásárláshoz külön új kulcsot generáljon?

**Comment by renced42** [comment]
Kedves @MicraMetripond 

A vevői alkalmazás minden egyes vásárláskor (QR Kód képzéskor) kulcspárt generál, függetlenül attól, hogy hányszor van újra telepítve.
Nem teljesen érthető a kérdés.
